process/smartContract/scQueryService_test.go

```diff
@@ -208,10 +208,9 @@ func TestExecuteQuery_WhenNotOkCodeShouldErr(t *testing.T) {
 
 	returnedData, err := target.ExecuteQuery(&query)
 
-	assert.NotNil(t, err)
-	assert.Contains(t, err.Error(), "error running vm func")
-	assert.Contains(t, err.Error(), "add more gas")
-	assert.Nil(t, returnedData)
+	assert.Nil(t, err)
```

Comment:
might rename the test(s) because they don't return errors any longer

Reply:
done